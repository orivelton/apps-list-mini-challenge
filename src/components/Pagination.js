import React from 'react';

const Pagination = () => {
  return (
    <ul className="pagination">
      <li><a href="/">&lt;</a></li>
      <li><a href="/">1</a></li>
      <li className="active"><a href="/">2</a></li>
      <li><a href="/">3</a></li>
      <li><a href="/">&gt;</a></li>
    </ul>
  )
};

export default Pagination;
