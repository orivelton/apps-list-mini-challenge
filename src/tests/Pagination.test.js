import React from 'react';
import renderer from 'react-test-renderer';
import Pagination from '../components/Pagination';

test('renders correctly', () => {
  const tree = renderer.create(<Pagination />).toJSON();
  expect(tree).toMatchSnapshot();
});
