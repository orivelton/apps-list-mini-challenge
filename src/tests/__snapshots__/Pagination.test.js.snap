// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<ul
  className="pagination"
>
  <li>
    <button
      disabled={false}
      onClick={[Function]}
    >
      &lt;
    </button>
  </li>
  <li>
    <button
      disabled={true}
      onClick={[Function]}
    >
      &gt;
    </button>
  </li>
</ul>
`;
